data_events.md

```diff
@@ -119,6 +119,21 @@ A user event ping includes some basic metadata (tab id, addon version, etc.) as
 }
 ```
 
+#### Sharing a site from toolbar
```

Comment:
Can you add a note here that this is a share *attempt* (i.e. the user clicks a share button), not necessarily a successful or failed share flow?